server/service/orbit_client.go

```diff
@@ -68,26 +127,115 @@ func (oc *OrbitClient) DoEnroll() (string, error) {
 	return resp.OrbitNodeKey, nil
 }
 
-func (oc *OrbitClient) GetConfig(orbitNodeKey string) (json.RawMessage, error) {
-	verb, path := "POST", "/api/fleet/orbit/config"
-	params := orbitGetConfigRequest{OrbitNodeKey: orbitNodeKey}
-	var resp orbitGetConfigResponse
-	err := oc.request(verb, path, params, &resp)
-	if err != nil {
-		return nil, err
+// enrollLock helps protect the enrolling process in case mutliple OrbitClients
+// want to re-enroll at the same time.
+var enrollLock sync.Mutex
```

Comment:
nit: `enrollMu`, also: should this be a field in the OrbitClient struct?

Reply:
Main purpose is to prevent concurrency issues if for some reason the Orbit app uses more than one `OrbitClient`.